osquery/tables/events/windows/windows_events.cpp

```diff
@@ -61,14 +61,25 @@ REGISTER(WindowsEventSubscriber, "event_subscriber", "windows_events");
 /// Helper function to recursively parse a boost ptree
 void parseTree(const pt::ptree& tree, std::map<std::string, std::string>& res) {
   for (const auto& node : tree) {
-    auto nodeName = node.second.get("<xmlattr>.Name", "");
+    // Skip this since it's not actually part of the EventData. Also prevents
+    // us from adding every Name attribute into its own key invalidly. This is
+    // part of a quirk of boost::ptree and its parsing of XML.
+    if (node.first == "<xmlattr>") {
+      continue;
+    }
 
+    auto nodeName = node.second.get("<xmlattr>.Name", "");
     if (nodeName.empty()) {
       nodeName = node.first.empty() ? "DataElement" : node.first;
     }
-    res[nodeName] = res[nodeName] == ""
-                        ? node.second.data()
-                        : res[nodeName] + "," + node.second.data();
+
+    if (res[nodeName] == "") {
+      res[nodeName] = node.second.data();
+    }
+    else {
```

Comment:
nit: bring this up to the closing bracket. We should install `clang-format` on your dev box, and you can auto lint with `clang-format -i`